react/what-is-props/what-is-props.md

```diff
@@ -0,0 +1,131 @@
+# What is props?
+
+---
+
+If you are beginner in react you must have heard about `props` a lot, so let's understand, how is it so important to understand.
+
+## How is it useful?
```

Comment:
You talked about React working in this section, let's try to give a more appropriate heading to this section